Jenkinsfile

```diff
@@ -23,7 +23,7 @@ def buildOs = 'linux'
 def buildJdk = '11'
 def buildMvn = '3.8.x'
 def runITsOses = ['linux']
-def runITsJdks = ['11', '17', '21']
+def runITsJdks = ['jdk_11_latest', 'jdk_17_latest', 'jdk_21_latest']
```

Comment:
We should update in https://github.com/apache/maven-jenkins-env/blob/master/vars/jenkinsEnv.groovy `jdkFromVersion`

Reply:
Ah, nice, I actually had no idea where those values were coming from :-)